main.js

```diff
@@ -0,0 +1,85 @@
+const inViewport = require('in-viewport');
+
+//find all convo cards
+const convoCards = document.getElementsByClassName('conversation');
+
+//for each card, define inViewport callback function
+[].slice.call(convoCards).forEach(card => inViewport(card, { offset: -200 }, function() {
```

Comment:
Agreed. What if you made a function to install this behavior that you passed into the forEach, like:

```
[].slice.call(convoCards).forEach(animateWhenVisible)
```